ndmg/scripts/ndmg_demo-mrilab

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env/python
```

Comment:
What is mrilab and why is it added here? That's not the point of this PR, if I understand.